gdal/.shed.yml

```diff
@@ -0,0 +1,85 @@
+categories:
+    - Ecology 
+description: Geospatial Data Abstraction Library tools are all dedicated to manipulate raster and vector geospatial data formats.
+long_description: |
+    Gdal is a translator library for raster and vector geospatial data formats that is released under an X/MIT style Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single raster abstract data model and single vector abstract data model to the calling application for all supported formats. It also comes with a variety of useful command line utilities for data translation and processing.
+
+homepage_url: https://www.gdal.org/
+name: gdal
+owner: mnhn65mo
```

Comment:
@yvanlebras what are we using here? `ecoteam`?

Reply:
For owner? ecoteam is fine. ecologyteam maybe just in case in the future there will be a Galaxy for economy... ;) Or biodivteam.

Reply:
Wait...wait...  Biodiversiteam ! Yes, that's it

Reply:
Nice ! i do this modif and also the remote_repository_url to match this repo  :)